tests/integration/update_cluster/minimal_gce/data/aws_s3_bucket_object_minimal-gce.example.com-addons-gcp-pd-csi-driver.addons.k8s.io-k8s-1.23_content

```diff
@@ -0,0 +1,828 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
should we use `{{ if WithDefaultBool .CloudConfig.ManageStorageClasses true }}` here? If someone want disable the default behaviour.